dace/codegen/CMakeLists.txt

```diff
@@ -412,6 +415,16 @@ endif() # DACE_ENABLE_RTL
 # Create Xilinx object files
 if(DACE_ENABLE_XILINX)
 
+  if (DACE_XILINX_TARGET_CLOCK MATCHES "|")
+    string(REGEX MATCH "0:([0-9]+)" DACE_XILINX_EXTERNAL_TARGET_CLOCK ${DACE_XILINX_TARGET_CLOCK})
```

Comment:
Apparently many of the CI tests errors are due to these new function (CMAKE errors)

Reply:
Yes! This has been fixed, as the regex did not match correctly before.